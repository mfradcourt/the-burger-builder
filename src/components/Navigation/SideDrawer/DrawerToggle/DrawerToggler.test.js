import React from 'react';
import { shallow } from "enzyme";

import DrawerToggle from './DrawerToggle';

describe('<DrawerToggle />', () => {
  let wrapper;

  it('should render DrawerToggle', () => {
    wrapper = shallow(
      <DrawerToggle/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
