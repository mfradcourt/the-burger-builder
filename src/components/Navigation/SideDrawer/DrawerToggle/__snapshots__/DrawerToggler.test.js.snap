// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DrawerToggle /> should render DrawerToggle 1`] = `
<div
  className="DrawerToggle"
>
  <div />
  <div />
  <div />
</div>
`;
